BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java

```diff
@@ -53,6 +54,7 @@ public Location(Location location) {
         this.latitude = location.latitude;
```

Comment:
Use `this(location.latitude, location. longitude, location. altitude);` and only set `this.timestamp` directly.